app/directives/forms/edit/edit-side-event.js

```diff
@@ -163,8 +165,10 @@ define(['app', 'lodash',
                         //============================================================
                         function loadConferences() {
                             return $http.get('/api/v2016/conferences', { 'params': findOpenRegsQuery() }).then(function(res) {
+                                const conferences = res.data.sort(compareDates).reverse();
+                                const { meetingId } = $route.current.params || {};
 
-                                $scope.options.conferences=res.data.sort(compareDates);
+                                $scope.options.conferences = isNeww() && meetingId? [_.find(conferences, { MajorEventIDs: [ meetingId ] })] : res.data.sort(compareDates).reverse();
```

Comment:
`isNeww` correct spelling

Reply:
yes I saw, thanks for calling out my laziness :-)

Reply:
@blaisefonseca resolved